source/dom/resolution.js

```diff
@@ -4,13 +4,11 @@
 		_riRatio,
 		_screenType,
 		_screenTypes = [ {name: 'standard', pxPerRem: 16, width: scope.innerWidth,  height: scope.innerHeight, aspectRatioName: 'standard'} ],	// Assign one sane value in case defineScreenTypes is never run.
-		_screenTypeObject;
+		_screenTypeObject,
+		_oldScreenType;
 
 	var getScreenTypeObject = function (type) {
 		type = type || _screenType;
-		if (type == _screenType && _screenTypeObject) {
-			return _screenTypeObject;
-		}
```

Comment:
Oh there was 2 reason why I remove it.
- if there is no parameter ({{type}}), type variable will be assigned as _screeType value and {{type == _screenType}} will be always true. But we can't assure _screeTypeObject is updated sync with screeType object.(it can return old resolution _screeTypeObject)
- this function seems to don't called very frequently so I thought it can have an array type search.

If second reason looks not reasonable, I'll change it as another way